### ISSUE TITLE
Heightmap generation problem.

### ISSUE THREAD

**Issue by T0FF23** [manticorp/unrealheightmap#15]
Hi, I encounter an error every time I generate a heightmap in the Alps area. No problem on the rest of the planisphere, and no problem generating an Albedo. Can you help me understand.
Thanking you in advance
Yours sincerely

![Uploading Heightmap_error.PNG…]()

**Comment by manticorp** [comment]
Sure :)

Are you able to provide a URL to a non working area please?

**Comment by T0FF23** [comment]
Hi, sorry but I don't understand your answer.
What URL should I provide?
Yours sincerely

**Comment by manticorp** [comment]
No worries!

When you have a view that you can't export, you should be able to copy the URL in the address bar of the browser, for example:

https://manticorp.github.io/unrealheightmap/#latitude/46.88647742351026/longitude/9.656982421875/zoom/9/outputzoom/15/width/8129/height/8129

Once you have a URL that doesn't work, if you can share it with me, I can check it as well and see why :)

I've done some exports of the alps myself, but I can't find any that don't work! It might be a particular location or zoom level.

**Comment by T0FF23** [comment]
Thank you very much for your answer which is perfectly clear to me now.
So here is the problematic URL :

https://manticorp.github.io/unrealheightmap/index.html#latitude/44.59046718130884/longitude/5.064697265625/zoom/7/outputzoom/11/width/8129/height/8129

This is all the more strange since it has worked very well in the past.
FYI, the problem occurs with the Chrome browser. Since then I tried with the Firefox browser and it works perfectly well.

Yours sincerely

**Comment by manticorp** [comment]
Hmm I've had no issues, are you able to try again?

![image](https://github.com/manticorp/unrealheightmap/assets/3750419/44770642-ffa9-48e6-a9b3-fdd0d4d066ad)
